guja-core/src/main/java/com/wadpam/guja/crud/CrudResource.java

```diff
@@ -46,7 +47,7 @@
  */
 @Consumes(value = {MediaType.APPLICATION_JSON})
 @Produces(CrudResource.MIME_JSON_UTF8)
-public class CrudResource<T, ID extends Serializable, D extends AbstractDao<T, ID>> {
+public class CrudResource<T, ID extends Serializable, D extends CrudDao<T, ID>> {
```

Comment:
I will still be able to access the "full" dao interface if I inherit from this class and implement additional methods?